en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -32,54 +42,219 @@ https://console.choreo.dev/login/</a>.
+              keepAliveTime = 600
+
+              [analytics.enforcer.LogReceiver.threadPool]
+                  coreSize = 10
+                  maxSize = 100
+                  #keep alive time of threads in seconds
+                  keepAliveTime = 600
+                  queueSize = 1000
+          ```
+
+### Step 2 - Try it out
+
+Follow the instructions below to generate some traffic in order to be able to view the Analytics in the Choreo cloud:
+
+1. Deploy your API.
+    
+     [Deploy your API in API Manager]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-api-in-choreo-connect/#step-3-deploy-the-api-in-api-manager) based on your setup.
+
+2. Invoke the API a few times.
+
+     For more information, see [Invoke the API]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-rest-api-in-choreo-connect/#step-4-invoke-the-api).
+
+3. Go to <a href="https://console.choreo.dev/insights" onclick="appInsights.trackEvent({name: 'apim-docs-choreo-connect-setup-api-analytics-choreo-insights'});">Choreo Insights</a> to view the statistics. 
+
+     Here are some of the graphs generated in the Choreo cloud.
+
+     [![Choreo Analytics Overview]({{base_path}}/assets/img/deploy/choreo-analytics-overview.png)]({{base_path}}/assets/img/deploy/choreo-analytics-overview.png)
+
+     [![Choreo Analytics Traffic]({{base_path}}/assets/img/deploy/choreo-analytics-traffic.png)]({{base_path}}/assets/img/deploy/choreo-analytics-traffic.png)
+
+     [![Choreo Analytics Latency]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)]({{base_path}}/assets/img/deploy/choreo-analytics-latency.png)
+
+##  Configuring ELK Analytics
+
+Follow the instructions below to configure ELK Analytics for Choreo Connect:
+
+[![Choreo ELK Analytics Data Flow]({{base_path}}/assets/img/deploy/choreo-elk-analytics-data-flow.png)]({{base_path}}/assets/img/deploy choreo-elk-analytics-data-flow.png)
+
+The Enforcer component in Choreo Connect can log analytics-related data to be used by Filebeat in ELK Stack. Data flow for the ELK Analytics can be depicted as given above.
+
+### Step 1 - Configure Choreo Connect
+
+#### Step 1.1 - Configure the config.toml file
+
+1. Open the `choreo-connect-1.x.x/docker-compose/choreo-connect(-with-apim)/conf/config.toml` file.
+2. Update the analytics section based on the following configurations.
+
+	The following configuration will enable the default reporter class (`org.wso2.am.analytics.publisher.reporter.elk.ELKMetricReporter`).
 
     ``` toml
     [analytics]
-    enabled = true
-    [analytics.adapter]
-        bufferFlushInterval = "1s"
-        bufferSizeBytes = 16384
-        gRPCRequestTimeout = "20s"
+       enabled = true
+       type = "ELK"
+          
     [analytics.enforcer]
+    [analytics.enforcer.configProperties]
+       "publisher.reporter.class" = "org.wso2.am.analytics.publisher.sample.reporter.CustomReporter"
```

Comment:
This should be removed in this config. It is only required in a custom reporter class as given in the below note section.

Reply:
Fixed in 981f2e98d09c63c41f5764612aac89cc5835f207